.travis.yml

```
@@ -39,14 +42,22 @@ stage_generic: &stage_generic
   script:
     # Compile the executables and run the tests.
     - python setup.py build_ext --inplace
-    - make test_ci
+    - export PYTHONHASHSEED=$(python -S -c "import random; print(random.  randint(1, 4294967295))")
+    - echo "PYTHONHASHSEED=$PYTHONHASHSEED"
+    - stestr run
```

If you want to just have the osx ci jobs run the one test failing without any overhead (including skipping test discovery) you can change this line to be:

```suggestion
    - stestr run -- -n test.python.pulse.test_commands.TestSamplePulse.test_pulse_limits
```